ucm2/sof-hda-dsp/Mics.conf

```diff
@@ -11,11 +49,12 @@ SectionDevice."HeadsetMic" {
 	}
 }
 
-SectionDevice."DigitalMic" {
+SectionDevice."Dmic" {
 	Comment "Digital Microphone"
 
 	ConflictingDevice [
-		"HeadsetMic"
+		"Headset Microphone"
```

Comment:
Wrong and wrong. It must be UCM device name not comment.

Reply:
the comment and name we're the same here (as I renamed headset microphone) device , but I will make device names and comments differ, so it is clear

Reply:
ok